packages/main/src/TableRow.js

```diff
@@ -80,6 +107,11 @@ class TableRow extends UI5Element {
 			return;
 		}
 
+		this.cells.forEach(cell => {
+			cell.removeEventListener("click", this.fnOnCellClick);
```

Comment:
you better fire some event from the cell and rely on bubbling so you can catch it in the Row

Reply:
Done